src/standalone/Soil/rre.jl

```diff
@@ -324,88 +326,81 @@ function ClimaLand.make_update_aux(model::RichardsModel)
             effective_saturation(ν, Y.soil.ϑ_l, θ_r),
         )
         @. p.soil.ψ = pressure_head(hydrology_cm, θ_r, Y.soil.ϑ_l, ν, S_s)
+
+        # If p.soil.dfluxBCdY is present, update it
```

Comment:
I dont think we need the if statement since we have a default `set_dfluxBCdY!` method for any model/any BC
I think this should not go in `update_aux!`, but instead here https://github.com/CliMA/ClimaLand.jl/blob/d60e2b27f0b87a37792d984ea08c7f6d810fb81d/src/standalone/Soil/rre.jl#L102

Reply:
because update_boundary_fluxes! is called after all the other cache variables are updated. In the soil/canopy (integrated model) case, the boundary flux will depend on cache variables in another component. if we need to account for dfluxBCdY in that case, we would need to set it here instead of in update_aux! to be sure the other cache variables are up to date.